google/cloud/aiplatform/initializer.py

```diff
@@ -75,6 +75,12 @@ def init(
             self._staging_bucket = staging_bucket
         if credentials:
             self._credentials = credentials
+        else:
+            logger = logging.getLogger("google.auth._default")
```

Comment:
Please move this logic into the `credentials` property to be consistent with the default project retrieval. Default `project` is pulled at time of accessing the property while, in this case, default `credential` is pulled when initializing the config.